src/pages/settings/Profile/Contacts/NewContactMethodPage.js

```diff
@@ -76,10 +77,10 @@ class NewContactMethodPage extends Component {
     */
     validateForm() {
         const login = this.state.login.trim();
-        const phoneLogin = LoginUtils.getPhoneNumberWithoutSpecialChars(login);
+        const phoneLogin = LoginUtils.appendCountryCode(LoginUtils.getPhoneNumberWithoutSpecialChars(login));
```

Comment:
We need to fix the bug here as mentioned in this comment https://github.com/Expensify/App/pull/17578#issuecomment-1516988970.

Reply:
I think that's enough since the bug is not technically on staging too.